src/metrics/metric_types.go

```diff
@@ -20,3 +24,61 @@ func (qd QueryDefinition) GetDataModels() interface{} {
 	ptr := reflect.New(reflect.ValueOf(qd.dataModels).Type())
 	return ptr.Interface()
 }
+
+func (qd *QueryDefinition) insertDatabaseNames(databases collection.DatabaseList) *QueryDefinition {
+	databaseList := ""
+	for database := range databases {
+		databaseList += `'` + database + `',`
+	}
+	databaseList = databaseList[0 : len(databaseList)-1]
```

Comment:
perhaps using some of the `Trim` functions on `strings`. It will improve readability and not panic if `len` is 0

Reply:
Fixed. Thx